source/features/copy-file.tsx

```diff
@@ -22,18 +23,29 @@ function handleClick({delegateTarget: button}: delegate.Event<MouseEvent, HTMLBu
 }
 
 function renderButton(): void {
-	for (const button of select.all('.file-actions .btn, [data-hotkey="b"]')) {
-		button
-			.parentElement! // `BtnGroup`
-			.prepend(
-				<button
-					className="btn btn-sm tooltipped tooltipped-n BtnGroup-item rgh-copy-file"
-					aria-label="Copy file to clipboard"
-					type="button"
-				>
-					Copy
-				</button>
-			);
+	for (const button of select.all<HTMLAnchorElement>([
+		'.file-actions .btn[href*="/raw/"]', // `isGist`
+		'[data-hotkey="b"]'
+	])) {
+		const copyButton = (
+			<button
+				className="btn btn-sm tooltipped tooltipped-n BtnGroup-item rgh-copy-file"
+				aria-label="Copy file to clipboard"
+				type="button"
+			>
+				Copy
+			</button>
+		);
+		const hasRenderButtons = button.parentElement!.querySelector('.rendered');
+		if (pageDetect.isGist() && hasRenderButtons) {
+			button.before(copyButton);
+		} else {
+			button
+				.parentElement! // `BtnGroup`
+				.prepend(copyButton);
+		}
+
+		groupSiblings(button);
```

Comment:
The code expected a group to be there; let's look for it explicitly:

```suggestion
		const group = button.closest('.BtnGroup');
		if (group) {
			group.prepend(copyButton);
		} else {
			groupButtons([button, copyButton]);
		}
```